### DISCUSSION TITLE
Error udm-iptv with UDM 1.12.22 and MagentaTV

### DISCUSSION THREAD

**Discussion by WW1983** [fabianishere/udm-iptv#156] [Q&A]
Hello,

i cant run udm-iptv after Update to 1.12.22. I have this Error:
=== Configuration ===

WAN Interface: eth8

WAN VLAN: 5 (dev iptv)

WAN DHCP: true (options "-O staticroutes -V IPTV_RG")

WAN Ranges: 224.0.0.0/4 87.141.0.0/16 193.158.0.0/15

LAN Interfaces: br0 br5

IGMP Proxy quickleave disabled: false

IGMP Proxy debug: false

=== IP Link and Route ===

=== Service Logs ===

Aug 05 08:07:17 ubnt udm-iptv[21417]: Sending discover...

Aug 05 08:07:21 ubnt udm-iptv[21417]: udhcpc failed to get a DHCP lease

Aug 05 08:07:21 ubnt udm-iptv[21417]: No lease, forking to background

Aug 05 08:07:21 ubnt udm-iptv[21417]: NATing IPTV network ranges (if necessary)

Aug 05 08:07:21 ubnt udm-iptv[21417]: Setting up igmpproxy

Aug 05 08:07:21 ubnt udm-iptv[21417]: Starting igmpproxy

Aug 05 08:07:21 ubnt igmpproxy[21417]: There must be at least 1 Vif as upstream.

Aug 05 08:07:21 ubnt systemd[1]: udm-iptv.service: Main process exited, code=exited, status=255/EXCEPTION

Aug 05 08:07:21 ubnt systemd[1]: udm-iptv.service: Failed with result 'exit-code'.

Aug 05 08:07:21 ubnt systemd[1]: udm-iptv.service: Consumed 132ms CPU time.

Any idea was i doing wrong? Are the firewall settings wrong?

Thx

**Comment by fabianishere** [comment]
You are not getting an IP address from your ISP for the IPTV network. Are you sure the config is still correct?

**Reply by WW1983** [reply]
The udm-iptv settings should be correct. But I'm not 100% sure. Can it be due to the firewall?

**Reply by fabianishere** [reply]
It has worked before, right?

**Reply by WW1983** [reply]
Yes, but I've tried so many changes, I don't remember what I changed

**Reply by WW1983** [reply]
I tryd now the WAN Interface "ppp0", but the same Error.

**Reply by WW1983** [reply]
Had the wrong settings. It works now. thx
